crypto/fipsmodule/ec/ecp_nistz256.c

```diff
@@ -196,6 +200,143 @@ void GFp_nistz256_point_add(P256_POINT *r, const P256_POINT *a, const P256_POINT
   limbs_copy(r->Y, res_y, P256_LIMBS);
   limbs_copy(r->Z, res_z, P256_LIMBS);
 }
+
+/* Include more reference implementation code for arches lacking assembly
+ * optimizations */
```

Comment:
Whatever code you copied out of gfp_p384.c must go into gfp_p256. Or, put another way, when you copy code out of a file with license X, you must copy it into a file that also has license X. ecp_nistz256.c and gfp_*.c have different licenses, and that's why they are split up in the weird way they are.

Probably the best thing to do w.r.t. this NIST curve code is to refactor it (in a separate PR) so that the P-256 code has the same shape as the P-384 code: rename ecp_nistz256.c to ecp_nistz256.inl, `#include "ecp_nistz256.inl"` in gfp_p256.c, etc.